react-native/android/build.gradle

```
@@ -242,7 +267,19 @@ def getNdkBuildFullPath() {
     return ndkBuildFullPath
 }
 
+def checkForHeadersFile(folder){
+    File[] files = new File("$projectDir/../../$folder").listFiles();
+
+    if (files == null || files.length == 0)
+        throw new GradleException("Didn't found any headers in the /$folder folder \n" +
+        "Please make sure you go to the root folder (realm-js) and do: \n git submodule update --init --recursive")
```

Just a suggestion to streamline wording..
```suggestion
        throw new GradleException("Unable to locate any headers in the '/$folder' folder.\n" +
        "Please make sure you go to the root folder (realm-js) and do:\n" +
        "git submodule update --init --recursive")
```